packages/babel-plugin/src/__tests__/module-traversal.test.tsx

```diff
@@ -283,19 +283,44 @@ describe('module traversal', () => {
     expect(result).toInclude('.cc-hash-test:hover{color:red;background-color:pink}');
   });
 
-  it('should reference property access expression from another module', () => {
+  it('should inline css from a directly called & assigned function mixin referencing an identifier from another module', () => {
     const result = transform(
       `
       import '@compiled/core';
       import { colorMixin } from '../__fixtures__/mixins/objects';
 
       const colors = colorMixin();
 
-      <div css={\`:hover { color: \${colors.color}; }\`} />
```

Comment:
i don't see a test remaining for this usage - can we add it back pls?

Reply:
🤔 You mean object and string literal tests along with this module traversal test?

Reply:
ohh got it. hehe. you mean this string literal test. 😛